src/common/configUtils.js

```diff
@@ -26,3 +24,7 @@ export function getDefaultConfig(host) {
 export function isProd(host) {
   return !!host.match('^ffiec')
 }
+
+export function isBeta(host) {
```

Comment:
Cypress was having an issue with the syntax of the CSS file imported in the `Beta` component.  I could not figure out why in the world that would be the case, so I had to just redefine the `isBeta` function.